src/stickyfill.js

```diff
@@ -98,8 +97,6 @@
       updateScrollPos();
       recalcAllPos();
     }
-
-    watchArray.forEach(forceRedraw);
```

Comment:
So this is no longer an issue?

![image](https://user-images.githubusercontent.com/27563/27585909-e79fe552-5af2-11e7-9ee0-f1cb8c208260.png)